src/tekton/taskrun.ts

```diff
@@ -40,4 +40,32 @@ export class TaskRun extends TektonItem {
     }
     return null;
   }
+
+  static async openDefinition(taskRun: TektonNode): Promise<void> {
+    const taskName = await TaskRun.getTaskNameByTaskRun(taskRun.getName());
```

Comment:
here should be an if condition to check taskRun is present.

Reply:
Done